tests/src/test/scala/system/basic/WskSequenceTests.scala

```diff
@@ -48,6 +49,26 @@ class WskSequenceTests extends TestHelpers with WskTestHelpers with StreamLoggin
 
   behavior of "Wsk Sequence"
 
+  it should "produce proper error when sequence component does not exist" in withAssetCleaner(wskprops) {
+    (wp, assetHelper) =>
+      val seqCompName = "seqComp"
+      val seqName = "seq"
+
+      wsk.action.create(seqCompName, Some(TestUtils.getTestActionFilename("echo.js")))
+
+      assetHelper.withCleaner(wsk.action, seqName) { (action, seqName) =>
+        action.create(seqName, Some(seqCompName), kind = Some("sequence"))
+      }
+
+      wsk.action.delete(seqCompName)
+
+      retry {
+        (withActivation(wsk.activation, wsk.action.invoke(seqName)) { activation =>
+          activation.response.result shouldBe Some(JsObject("error" -> sequenceComponentNotFound.toJson))
+        })
```

Comment:
Why is the retry needed here? `withActivation` already does the polling I think?